packages/experimental/Text/src/Text.tsx

```diff
@@ -73,11 +73,11 @@ export const Text = compose<TextType>({
         style: {
           margin: 0,
           color: tokens.color,
-          ...fontStyles.from(tokens, theme)
-        }
+          ...fontStyles.from(tokens, theme),
+        },
       }),
-      ['color', ...fontStyles.keys]
-    )
+      ['color', ...fontStyles.keys],
+    ),
```

Comment:
Did you run prettier? I see a lot of extraneous commas throughout

Reply:
We were getting inconsistency on this setting, after a quick poll in the teams channel we decided to make trailing commas default and I recently updated our general settings to use that. As a result we'll see a bunch of changes like this as people edit files.  Long term though it should sort itself out.

I'm tempted to do a full pass but it will generate a lot of noise for people with changes in progress.